Pcap++/header/PcapDevice.h

```diff
@@ -18,6 +18,75 @@ namespace pcpp
 	// Forward Declaration - required for IPcapDevice::matchPacketWithFilter
 	class GeneralFilter;
 
+	namespace internal
+	{
+		/// @class PcapHandle
+		/// @brief A wrapper class for pcap_t* which is the libpcap packet capture descriptor.
+		/// This class is used to manage the lifecycle of the pcap_t* object
+		class PcapHandle
+		{
+		public:
+			/// @brief Creates an empty handle.
+			constexpr PcapHandle() noexcept = default;
+			/// @brief Creates a handle from the provided pcap descriptor.
+			/// @param pcapDescriptor The pcap descriptor to wrap.
+			explicit PcapHandle(pcap_t* pcapDescriptor) noexcept;
+
+			PcapHandle(const PcapHandle&) = delete;
+			PcapHandle(PcapHandle&& other) noexcept;
+
+			PcapHandle& operator=(const PcapHandle&) = delete;
+			PcapHandle& operator=(PcapHandle&& other) noexcept;
+			PcapHandle& operator=(std::nullptr_t) noexcept;
+
+			~PcapHandle();
+
+			/// @brief Check if the handle is not null.
+			/// @return True if the handle is not null, false otherwise.
```

Comment:
If both `@brief` and `@return` contain the same information, we usually omit `@brief` and keep only `@return`.

Same for the other methods in this class

Reply:
Fixed e481d1c and febc6f4